mps_youtube/helptext.py

```diff
@@ -79,6 +79,8 @@ def helptext():
     {2}dvupl <username>{1} - download user's YouTube playlists (video).
     {2}dlurl <url or id>{1} download a YouTube video by url or video id.
     {2}playurl <url or id>{1} play a YouTube video by url or id.
+    {2}browserplay <url or id>{1} play a YouTube video by url or id in browser.
```

Comment:
browserplay was intended for opening a given ID or URL. Since it points to a single video, I'm not really sure how the number would help. I can work on adding this functionality to browsersearch, though, which opens the search query in the browser.